src/renderer/components/message/MessageWrapper.tsx

```diff
@@ -64,8 +68,13 @@ export function MessageWrapper(props: RenderMessageProps) {
   ])
 
   return (
-    <li id={props.key2} className='message-wrapper'>
-      <Message {...props} />
+    <li
+      id={props.key2}
+      className={classNames('message-wrapper', {
+        'overflow-anchor': reactionBarShownForMessageId === props.message.id,
```

Comment:
This is made to not hide the reactions bar when a new message arrives.
But there is still an issue with this. If you open a reactions bar and someone leaves a reaction on a message that is above it, the reactions bar will hide.